src/main/java/be/seeseemelk/mockbukkit/MockUnsafeValues.java

```diff
@@ -296,32 +301,121 @@ public Entity deserializeEntity(byte[] data, World world, boolean preserveUUID)
 		throw new UnimplementedOperationException();
 	}
 
+	/**
+	 * Gets the translation key for a {@link Material} that {@link Material#isBlock() is a block}.
+	 * @param material the material to translate
+	 * @return a string of the structure "block.{@literal <namespace>}.{@literal <material>}" (e.g. "block.minecraft.stone"), or null if not a block.
+	 */
```

Comment:
You don't need to add a comment for the overriden methods.

Reply:
Thought I'd "publicly" share the way I faked the vanilla behaviour, but if it's against your style guidelines I can remove them

Reply:
There's no guidline for this, but it seems unnecessary.